zebrad/proto/compact_formats.proto

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2019-2020 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+syntax = "proto3";
+package cash.z.wallet.sdk.rpc;
```

Comment:
I looked at the repository history, and it seems like the two files have infrequent updates ([`service.proto`](https://github.com/zcash/lightwalletd/commits/master/walletrpc/service.proto) and [`compact_formats.proto`](https://github.com/zcash/lightwalletd/commits/master/walletrpc/compact_formats.proto)), but it still could break.

My take is that since this is currently only used for testing, we're likely to find out if there are any breaking changes that matter to us when the test fails :thinking: We might want to revisit this if we start using it in other places (maybe with a CI job that downloads the two files from the remote repository and checks if they are the same as the ones in our repository).